### ISSUE TITLE
doesn't work with on Internet Explorer

### ISSUE THREAD

**Issue by siemya** [anorudes/React-Scroll-Effect#6]
İt works great on chrome, but doesn't work on IE. Nothing appears.

```
<ScrollEffect animate="fadeIn" duration="2" offset="-400">
   {this.props.children}
</ScrollEffect>
```

**Comment by CoreyBurkhart** [comment]
Looks like window.scrollY is [not supported](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollY) in IE. 

`scrollPositionY = window.scrollY,`
should be 
`scrollPositionY = window.pageYOffset` for IE 9+ support

**Comment by anorudes** [comment]
Solved in last pr #5
